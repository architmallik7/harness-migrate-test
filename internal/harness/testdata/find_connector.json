{
    "status": "SUCCESS",
    "data": {
        "connector": {
            "name": "pranavgcs",
            "identifier": "pranavgcs",
            "description": "",
            "orgIdentifier": "default",
            "projectIdentifier": "pranavt1",
            "tags": {},
            "type": "Gcp",
            "spec": {
                "credential": {
                    "type": "ManualConfig",
                    "spec": {
                        "secretKeyRef": "pranavgcs"
                    }
                },
                "delegateSelectors": [],
                "executeOnDelegate": false
            }
        },
        "createdAt": 1671561889912,
        "lastModifiedAt": 1671561889906,
        "status": {
            "status": "SUCCESS",
            "errorSummary": null,
            "errors": null,
            "testedAt": 1671561890282,
            "lastTestedAt": 0,
            "lastConnectedAt": 1671561890282
        },
        "activityDetails": {
            "lastActivityTime": 1671561889931
        },
        "harnessManaged": false,
        "gitDetails": {
            "objectId": null,
            "branch": null,
            "repoIdentifier": null,
            "rootFolder": null,
            "filePath": null,
            "repoName": null,
            "commitId": null,
            "fileUrl": null,
            "repoUrl": null
        },
        "entityValidityDetails": {
            "valid": true,
            "invalidYaml": null
        },
        "governanceMetadata": null
    },
    "metaData": null,
    "correlationId": "d4bacf7d-9171-4972-8782-59e61b060aae"
}