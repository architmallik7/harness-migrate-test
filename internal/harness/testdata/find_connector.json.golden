{
  "name": "pranavgcs",
  "identifier": "pranavgcs",
  "orgIdentifier": "default",
  "projectIdentifier": "pranavt1",
  "type": "Gcp",
  "spec": {
    "credential": {
      "spec": {
        "secretKeyRef": "pranavgcs"
      },
      "type": "ManualConfig"
    },
    "delegateSelectors": [],
    "executeOnDelegate": false
  }
}